<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QRadioButton" name="clockwise180RadioButton">
     <property name="text">
      <string>rotate 180° clockwise</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QRadioButton" name="byAngleRotationRadioButton">
     <property name="text">
      <string>rotate clockwise by angle</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="noRotationRadioButton">
     <property name="text">
      <string>no rotation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="counterClockwiseRadioButton">
     <property name="text">
      <string>rotate counterclockwise</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QRadioButton" name="clockwiseRotationRadioButton">
     <property name="text">
      <string>rotate clockwise</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QSpinBox" name="angleSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="clockwiseLabel">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="rotate.qrc">:/plugins/rotate/clockwise</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="counterClockwiseLabel">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="rotate.qrc">:/plugins/rotate/counterclockwise</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rotate.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>byAngleRotationRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>angleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
